index.html

```diff
@@ -0,0 +1,128 @@
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+
+  <link rel="stylesheet" href="./Components/Pages/Styles/main.css">
+  <title>Game Ranking: Home page</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navbar navbar-expand-md navbar-light bgp-light">
+      <div class="container-fluid">
+        <a class="navbar-brand p-0 m-0" href="#"><img class="p-0 m-0 logo-container"
+            src="./Components/Pages/Assets/icon-game.gif" alt="icon"></a>
+        <button class="navbar-toggler me-auto" type="button" data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+          aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page"
+                href="./Components/Pages/About/Component/about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">News</a>
+            </li>
+            <li class="nav-item mx-5 nav-content">
+              <a class="nav-link">CC Campain</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div class="body-bg d-flex justify-content-center">
+      <div class="header">
+        <video class="d-flex justify-content-center" id="background-video" autoplay loop muted
+          poster="./Components/Pages/Assets/mkke-bg.png">
+          <source src="./Components/Pages/Assets/mkkefinal.mp4" type="video/mp4">
+        </video>
+        <div class="title d-flex justify-content-center flex-column">
+          <h4>"Hello! Sharing world"</h4>
+          <h1>RANKING OF VIDEO GAMES IN 2021</h1>
+          <div class="descruption">
+            <p>Games are more than just games. It's odd phrasing but it's the truth. They are meant to enrapture, encapture, pull in a player and audience through three methods: visuals, gameplay, and narrative. we are going to list best game ranking in the world with some details according to testimonies of players
+            </p>
+          </div>
+          <div class="prod-year">
+            <h3>
+              2019.10.15 (Monday) ~ 16(Tuesday)
+            </h3>
+            <p>@informations share by regiss mukubiza</p>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] On mobile view these sections are overlapping each other, please make sure to not use fixed heights/widths to prevent this from happening.

![](https://i.imgur.com/GVWiiNA.png)

Reply:
Addressed 😎